share/instruments/instruments.xml

```diff
@@ -3600,13 +3600,13 @@
                         <program value="22"/> <!--Harmonica-->
                   </Channel>
             </Instrument>
-            <Instrument id="harmonica-d12-g">
+            <Instrument id="harmonica-d10g">
```

Comment:
@oktophonie, I took the opportunity to remove the dash before the "g" in this ID to make it consistent with the others.